src/plrmodes/marisa_b.c

```diff
@@ -104,9 +104,12 @@ TASK(marisa_star_slave_projectile, {
 }) {
 	MarisaBController *ctrl = ARGS.ctrl;
 	Player *plr = ctrl->plr;
+	MarisaBSlave *slave = ENT_UNBOX(ARGS.slave);
 
-	if(!ENT_UNBOX(ARGS.slave)->alive || !player_should_shoot(plr)) {
-		return;
+	if(slave) {
+		if(!slave->alive || !player_should_shoot(plr)) {
+			return;
+		}
```

Comment:
This one can/should just be `NOT_NULL(ENT_UNBOX(ARGS.slave))`. That'll throw an assertion error if the task happens to be called without a slave reference.

Note that this doesn't apply to the `ENT_UNBOX` inside the for loop later in this function.